src/github/pullRequestOverview.ts

```diff
@@ -340,30 +340,74 @@ export class PullRequestOverviewPanel {
 		}
 	}
 
+	private getReviewersQuickPickItems(assignableUsers: IAccount[], suggestedReviewers: SuggestedReviewer[]): vscode.QuickPickItem[] {
+		// used to track logins that shouldn't be added to pick list
+		// e.g. author, existing and already added reviewers
+		const skipList: Set<string> = new Set([
+			this._pullRequest.author.login,
+			...this._existingReviewers.map(reviewer => reviewer.reviewer.login)
+		]);
+
+		const reviewers: vscode.QuickPickItem[] = [];
+		for (const { login, name, isAuthor, isCommenter } of suggestedReviewers) {
+			if (skipList.has(login)) {
+				continue;
+			}
+
+			const suggestionReason: string =
+				isAuthor && isCommenter
+					? 'Recently edited and reviewed changes to these files'
+					: isAuthor
+						? 'Recently edited these files'
+						: isCommenter
+							? 'Recently reviewed changes to these files'
+							: 'Suggested reviewer';
+
+			reviewers.push({
+				label: login,
+				description: name,
+				detail: suggestionReason
+			});
+			// this user shouldn't be added later from assignable users list
+			skipList.add(login);
+		}
+
+		for (const { login, name } of assignableUsers) {
+			if (skipList.has(login)) {
+				continue;
+			}
+
+			reviewers.push({
+				label: login,
+				description: name
+			});
+		}
+
+		return reviewers;
+	}
+
 	private async addReviewers(message: IRequestMessage<void>): Promise<void> {
 		try {
-			const allMentionableUsers = await this._pullRequestManager.getMentionableUsers();
-			const mentionableUsers = allMentionableUsers[this._pullRequest.remote.remoteName];
-			const newReviewers = mentionableUsers
-				.filter(user =>
-					!this._existingReviewers.some(reviewer => reviewer.reviewer.login === user.login)
-					&& user.login !== this._pullRequest.author.login);
-
-			const reviewersToAdd = await vscode.window.showQuickPick(newReviewers.map(reviewer => {
-				return {
-					label: reviewer.login,
-					description: reviewer.name
-				};
-			}), {
-				canPickMany: true,
-				matchOnDescription: true
-			});
+			const [allAssignableUsers, suggestedReviewers] = await Promise.all([
+				this._pullRequestManager.getAssignableUsers(),
+				this._pullRequestManager.getSuggestedReviewers(this._pullRequest)
+			]);
+			const assignableUsers = allAssignableUsers[this._pullRequest.remote.remoteName];
+
+			const reviewersToAdd = await vscode.window.showQuickPick(
+				this.getReviewersQuickPickItems(assignableUsers, suggestedReviewers),
+				{
+					canPickMany: true,
+					matchOnDescription: true
+				}
+			);
 
 			if (reviewersToAdd) {
 				await this._pullRequestManager.requestReview(this._pullRequest, reviewersToAdd.map(r => r.label));
 				const addedReviewers: ReviewState[] = reviewersToAdd.map(reviewer => {
 					return {
-						reviewer: newReviewers.find(r => r.login === reviewer.label)!,
+						// assumes that suggested reviewers will be a subset of assignable users
+						reviewer: assignableUsers.find(r => r.login === reviewer.label)!,
```

Comment:
From what I've seen in a few repositories - suggested reviewers always were present in assignable users list.
Are there cases when it won't be true?

Reply:
I think this is a safe assumption. I just tried removing a collaborator who was previously a suggested reviewer and they were removed from both lists